examples/gardener/main.go

```diff
@@ -13,10 +13,10 @@ import (
 )
 
 func main() {
-	projectName := flag.String("p", "", "Gardener project name")
+	projectName := flag.String("p", "garden-omar", "Gardener project name")
```

Comment:
Please remove any personal or private data from the code.

Reply:
removed